components/calendar/index.js

```diff
@@ -7,7 +7,9 @@ import { API_EVENTS_URL } from '../../common/constants';
 require('es6-promise').polyfill();
 require('isomorphic-fetch');
 
-const TODAY = moment()
+moment.locale('nb');
```

Comment:
Sure, fixed that now. Seems a bit weird to import moment to the index at root just to set locale, but sure.